README.md

```diff
@@ -5,6 +5,7 @@
 
 - [@gdams](https://github.com/gdams) - George Adams
 - [@pnstanton](https://github.com/pnstanton) - Peter Stanton
+- [@johnoliver](https://github.com/johnoliver) - John Oliver
```

Comment:
please move this up one to keep alphabetical order and also add him to the infrastructure group below